CharacterCreation/06_01_Systemized_Skills.txt

```diff
@@ -1,68 +1,125 @@
 
-* Acrobatics
-* Athletics
-* Deception
-* Insight
-* Intimidation
-* Investigation
-* Medical
-* Navigation
-* Space
-* Geography
-* Persuasion
-* Repair
-* Sleight of Hand 
-* Spotting
-* Stealth
-* Survival
+	Whenever you roll to see if your character succeeds in a task they have 
+skill in, add both your character's points in relevant skills and the relevant
+stat modifier. Ask your GM to confirm what skills and stats apply to the roll.
+
+===== Gaining Skill Points and Skill Proficiency =====
+	Each time a character levels up, they gain a certain number of skill
+points to distribute as they choose. However, a character may only gain up to 5
+points in a skill per level, up to a maximum of 60. Some feats, backgrounds and
+class passives will give a character points in a specific skill. While these
+skill gains are not subjected to the limit on skill point gain per level, they
+do contribute to the maximum of 60 points in a skill.
+
+	Characters are considered to be proficient with certain skills. When a
+character is proficient in a skill, each skill point they spend in that skill
+upon leveling up increases that skill by two instead of one.
+
+===== Skill Checks and Critical Successes =====
+	If you roll a critical success on a skill check, add 10 to your total roll.
+For an extra critical, add 20.
+
+===== "General" Skills =====
+	Some skill groups such as Knowledge, Weapons, and Vehicles allow
+characters to learn the skill both in general and in specific cases. If the
+character does not know the specific skill in a group that applies to a situation,
+the player can instead add half of the character's general skill in that skill
+group (rounded down). For example, if a GM asks for a roll in Knowledge: History,
+but the character only has 10 skill in Knowledge: General, the player can add 5
+from Knowledge: General to the Knowledge: History roll. However, if the  GM asks
+for a roll in Knowledge: General, the player can all all 10 points from
+Knowledge: General to the roll.
 
+===== Skills in Compound X =====
+	This list of some of the most commonly-used skills in Compound X is
+arranged based on the stats that are usually associated with the skill. Your GM
+may decide that a different stat applies to any given instance. In addition,
+this is not an exhaustive list of skills, merely the skills called for most
+often. Talk to your GM if you would like your character to learn a more specific
+case of one of these skills or a skill not listed below.
+
+== Charisma ==
+Deception:
+	To tell a convincing lie
+Insight:
+	To tell truth and lies apart
+Performance:
+	"Roll for interpretive dance"
+Persuasion:
+	To tell the truth, share an opinion, or change someone's mind
+
+== Dexterity ==
+Acrobatics:
+	"PARKOUR!"
+Sleight of Hand:
+	To pick pockets, pull off card tricks, and cause other mayhem
+Stealth:
+	To move quietly and escape notice
 Vehicles:
-* GENERAL*
+	For checks to pilot vehicles
+* General
 * Flight

```

Comment:
It seems more likely to me that players will be flying ships then helicopters. Can we have this be Space and collapse air vehicles into general instead?

Reply:
I disagree on this. Not only are there several dungeons with helicopters in them, I want to give room for other planet-side dungeons.
I'm willing to talk splitting air and space vehicles into separate skills, but not completely disregarding air vehicles as a specific case.
In conclusion, I'm going to boot this discussion to a different pr and/or new issue.

Reply:
Entirely possible I have a bias towards space exacerbated by never having participated in a dungeon with a helicopter.

Reply:
If you want a Vehicles: Space skill, make an issue.